pkgs/development/tools/build-managers/bazel/bazel_7/cpp-test.nix

```diff
@@ -24,8 +24,8 @@ let
     # Take all the rules_ deps, bazel_ deps and their transitive dependencies,
     # but none of the platform-specific binaries, as they are large and useless.
     requiredDepNamePredicate = name:
-      null == builtins.match ".*(macos|osx|linux|win|apple|android|maven).*" name
-      && null != builtins.match "(platforms|com_google_|protobuf|rules_|bazel_).*" name ;
+      null == builtins.match ".*(macos|osx|linux|win|android|maven).*" name
+      && null != builtins.match "(platforms|com_google_|protobuf|rules_|bazel_|apple_support).*" name;
```

Comment:
This might not filter all platform specific deps with `apple` in the name. I needed this to allow downloading of `apple_support`. Suggestions welcome.

Reply:
Having too much here is not a big issue. It bloats the build closure, but not the runtime closure. So pretty innocuous.